core/block/simple/table/utils.go

```diff
@@ -0,0 +1,9 @@
+package table
+
+import "strings"
+
+const TableCellSeparator = "-"
+
+func IsTableCell(blockId string) bool {
+	return strings.Contains(blockId, TableCellSeparator)
```

Comment:
maybe do more strict check? Cells have exactly this shape: `bsonHex-bsonHex`, so check like this should work well:
```
tokens := strings.Split(blockId, separator)
return len(tokens) == 2
```

Reply:
> maybe do more strict check
Shall we?
We've had a bug with wrongly generated ids some time before, which includes uuid and IDs with more than two delimiters.
I've decided to to establish a rule where any ID not related to table cells can not contain any delimiters.
If it does, I don't consider it a proper Id, and then all the content will be pasted as if it were in a table cell.
It is never wrong, and even if we somehow encounter an incorrect table cell id with more than 2 delimiters everything will work just fine.

Reply:
cell id has special meaning: rowId + columnId